meetpy/meetpy/urls.py

```diff
@@ -6,7 +6,7 @@
 admin.autodiscover()
 
 urlpatterns = [
-    url(r'^', include('misc.urls', namespace='misc')),
-    url(r'^', include('meetups.urls', namespace='meetups')),
-    url(r'^admin/', include(admin.site.urls)),
+    url(r'^', include(('misc.urls', 'misc'), namespace='misc')),
```

Comment:
czy to działa? jak patrzę na tą funkcję to wydaje mi się, że coś tu nie gra:
https://docs.djangoproject.com/pl/2.1/ref/urls/#django.urls.include
```
include(module, namespace=None)
include(pattern_list)
include((pattern_list, app_namespace), namespace=None)
```

Reply:
wygląda na to, że działa, chyba dlatego bo django robi jeszcze https://github.com/django/django/blob/master/django/urls/conf.py#L33 ale faktycznie na przyszłośc można by to pozamieniać na normalne importy